### PR TITLE
Add wml sample

### PR DESCRIPTION
**Pull request by kevinyu98**
**Which issue is resolved by this Pull Request:** 
Resolves #

**Description of your changes:**

- Add wml train and serve example

- Update the default tensorflow version to 1.15. WML V4 api is no longer supporting 1.14

**Environment tested:**

* Python Version (use `python --version`): 3.6.4
* Tekton Version (use `tkn version`): 0.11.3
* Kubernetes Version (use `kubectl version`): 1.16
* OS (e.g. from `/etc/os-release`):

### CODE REVIEW

**Comment by kevinyu98** [comment]
@Tomcli

File: samples/watson-train-serve/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+- Install [Watson Requirements](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/ibm-samples/watson#requirements) 
+
+## Instructions
+
+1. Compile and apply the trusted-ai pipeline
+```shell
+dsl-compile-tekton --py watson_train_serve_pipeline.py --output watson-train-server.yaml
+kubectl apply -f watson-train-server.yaml 
+```
+2. If your service account hasn't have RBAC, follow this [sa-and-rbac](/sdk/sa-and-rbac.md) to setup.
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:23]:
```suggestion
2. If your default service account doesn't have edit permission, follow this [sa-and-rbac](/sdk/sa-and-rbac.md) to setup.
```

File: samples/watson-train-serve/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+3. Run the kfp-on-wml-training pipeline, click the `enter` key to use the default pipeline variables except these two variables,
+`GITHUB_TOKEN`: your github token
+`CONFIG_FILE_URL`: your configuration file which stores the credential information, here is the example of [creds.ini file](https://github.com/kubeflow/pipelines/blob/master/samples/contrib/ibm-samples/watson/credentials/creds.ini) 
+
+```shell
+tkn pipeline start kfp-on-wml-training --showlog
+```
+
+This pipeline will run for 5 to 10 minutes, then you should able to see the result at the end of the logs.
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:33]:
```suggestion
This pipeline will run for 5 to 10 minutes, then you should able to see the prediction result `{'values': [5, 4]}` at the end of the logs.
```

File: samples/watson-train-serve/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+## Instructions
+
+1. Compile and apply the trusted-ai pipeline
+```shell
+dsl-compile-tekton --py watson_train_serve_pipeline.py --output watson-train-server.yaml
+kubectl apply -f watson-train-server.yaml 
+```
+2. If your service account hasn't have RBAC, follow this [sa-and-rbac](/sdk/sa-and-rbac.md) to setup.
+
+3. Run the kfp-on-wml-training pipeline, click the `enter` key to use the default pipeline variables except these two variables,
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:25]:
```suggestion
3. Run the kfp-on-wml-training pipeline, click the `enter` key to use the default pipeline variables except for these two variables,
```

File: samples/watson-train-serve/watson_train_serve_pipeline.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+    wml_deploy = deploy_op(
+                  wml_store.output,
+                  model_name,
+                  scoring_payload
+                 ).apply(params.use_ai_pipeline_params(secret_name)).set_image_pull_policy('Always')
+
+if __name__ == '__main__':
+    # compile the pipeline
+    from kfp_tekton.compiler import TektonCompiler
+    TektonCompiler().compile(flipcoin_pipeline, __file__.replace('.py', '.yaml'))
```
Comment (Tomcli) [inline samples/watson-train-serve/watson_train_serve_pipeline.py:90]:
```suggestion
    TektonCompiler().compile(kfp_wml_pipeline, __file__.replace('.py', '.yaml'))
```

File: samples/watson-train-serve/README.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+        ```shell
+        kubectl patch cm feature-flags -n tekton-pipelines -p '{"data":{"disable-home-env-overwrite":"true","disable-working-directory-overwrite":"true"}}'
+        ```
+- Install [kfp-tekton](/sdk/README.md#steps) SDK
+
+- Install [Watson Requirements](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/ibm-samples/watson#requirements) 
+
+## Instructions
+
+1. Compile and apply the trusted-ai pipeline
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:18]:
```suggestion
1. Compile and apply the Watson ML pipeline
```

File: samples/watson-train-serve/README.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
+# Watson Train and Server.
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:1]:
```suggestion
# Training and Serving Models with Watson Machine Learning
```

File: samples/watson-train-serve/README.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+This pipeline runs training, storing and deploying a Tensorflow model with MNIST handwriting recognition using [IBM Watson Machine Learning](https://www.ibm.com/cloud/machine-learning) service.
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:3]: maybe add an extra sentence to point to the original example, so the users know what's going on with this example.

File: samples/watson-train-serve/README.md (hunk 1 above)
Comment (kevinyu98) [reply samples/watson-train-serve/README.md:23]: thanks.

File: samples/watson-train-serve/README.md (hunk 2 above)
Comment (kevinyu98) [reply samples/watson-train-serve/README.md:33]: done

File: samples/watson-train-serve/README.md (hunk 3 above)
Comment (kevinyu98) [reply samples/watson-train-serve/README.md:25]: ok

File: samples/watson-train-serve/watson_train_serve_pipeline.py (hunk 4 above)
Comment (kevinyu98) [reply samples/watson-train-serve/watson_train_serve_pipeline.py:90]: done

File: samples/watson-train-serve/README.md (hunk 5 above)
Comment (kevinyu98) [reply samples/watson-train-serve/README.md:18]: ok

File: samples/watson-train-serve/README.md (hunk 6 above)
Comment (kevinyu98) [reply samples/watson-train-serve/README.md:1]: ok

File: samples/watson-train-serve/README.md (hunk 7 above)
Comment (kevinyu98) [reply samples/watson-train-serve/README.md:3]: sure

File: samples/watson-train-serve/README.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
+# Training and Serving Models with Watson Machine Learning
+
+This pipeline runs training, storing and deploying a Tensorflow model with MNIST handwriting recognition using [IBM Watson Machine Learning](https://www.ibm.com/cloud/machine-learning) service. And it is coming from [ibm-samples/watson](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/ibm-samples/watson).
```
Comment (Tomcli) [inline samples/watson-train-serve/README.md:3]:
```suggestion
This pipeline runs training, storing and deploying a Tensorflow model with MNIST handwriting recognition using [IBM Watson Machine Learning](https://www.ibm.com/cloud/machine-learning) service. This example is originated from Kubeflow pipeline's [ibm-samples/watson](https://github.com/kubeflow/pipelines/tree/master/samples/contrib/ibm-samples/watson) example.
```

Comment (kevinyu98) [reply samples/watson-train-serve/README.md:3]: ok

**Pull request merged**